API.md

```diff
@@ -188,10 +192,10 @@ ListNavigation
 ![Component Image](./docs/ListNavigation.png)
 
 
-Prop | Required | Default | Type | Description 
```

Comment:
Is this white space change because you opened the file in your editor and then saved it?

Perhaps fix the white space changes in the generator script too, so that next time someone generates this file it doesn't put all the white space back in again?

Reply:
yeah